packages/lab/src/chat/ChatMessage.tsx

```diff
@@ -13,6 +14,7 @@ interface ChatMessageProps {
 
 export const ChatMessage = forwardRef<HTMLDivElement, ChatMessageProps>(
   ({ author, role, dateTime, text, variant }, ref) => {
```

Comment:
Can we change prop signature for dateTime?

So we should receive Date | DateTime | string. Like all inputs `formatDate` recieves

Reply:
fixed